oereb_client/static/src/component/map_query/map_query.js

```diff
@@ -69,7 +69,7 @@ const OerebMapQuery = function (props) {
 
   const listResults = mapQuery.results.map((result, key) => {
     const egrid = result.egrid;
-    const identdn = result.identdn;
+    const identdn = result.identDN;
```

Comment:
I wonder if it would be clearer if you also changed the name of your variables?

Reply:
The problem is that there is no consistent definition in the specification (`IDENTDN`, `identDN`, `IdentDN`, ...)